### PR TITLE
Add one, update 2 publications

### PR DESCRIPTION
**Pull request by cranmer**

### CODE REVIEW

File: _data/publications/1752736.yml [hunk 1]
```diff
@@ -2,4 +2,7 @@ inspire-id: 1752736
 project: 
 - madminer
 - exploratory-ml
+link: https://doi.org/10.3847/1538-4357/ab4c41
+date: November 19, 2019
```
Comment (henryiii) [inline _data/publications/1752736.yml]: Dates in ISO 8061 format, please
```suggestion
date: 2019-11-19
```

File: _data/publications/1675782.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+inspire-id: 1675782
+project: 
+- madminer
+- exploratory-ml
+link: https://doi.org/10.1073/pnas.1915980117
+date: February 20, 2020
```
Comment (henryiii) [inline _data/publications/1675782.yml]:
```suggestion
date: 2020-01-20
```

File: _data/publications/1752736.yml (hunk 1 above)
Comment (cranmer) [reply _data/publications/1752736.yml]: Ok, should probably update instructions. Example is `date: Dec 10, 2018`
https://iris-hep.org/docs/add_publication.html

Also, would be nice if there was an example for how the publication is rendered. Not always clear

Comment (henryiii) [reply _data/publications/1752736.yml]: I had a sneaking suspicion that the example was probably wrong...

Comment (henryiii) [reply _data/publications/1752736.yml]: What do you mean by an example of rendering? I'd be happy to add one.

Comment (cranmer) [reply _data/publications/1752736.yml]: I updated the file in the PR. I didn't see the "Commit suggestion" option until now. Both files updated.

**Comment by cranmer** [comment]
@henryiii it seems this format fails, while other format worked.

**Comment by cranmer** [comment]
trying with 
```date: '2020-02-20'```

**Comment by henryiii** [comment]
Looks like a bug in the part that reads the date from inspire; I think you are the first to try to override the date from inspire. It is mixing dates and strings. I should be able to fix that.

Why are you overriding the date and citation, out of curiosity?

**Comment by cranmer** [comment]
The papers are now published, but INSPIRE doesn't have the updated info.

**Comment by cranmer** [comment]
Ok, checks passed. Do you want to merge? Or I will if I don't hear from you in ~5 min. I'll check to see what gets rendered.

**Comment by henryiii** [comment]
Give me a couple of minutes to see if I can understand why it's failing when not a string, then we can merge.

**Comment by henryiii** [comment]
Nevermind, actually it's easier to debug if this is in master, I can just try removing the quotes. Let's merge.

**Pull request merged**